build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
As per my previous comment, favour portability.
```suggestion
#!/bin/sh
```